cdap-ui/app/cdap/components/FastAction/SendEventAction/SendEventAction.less

```diff
@@ -36,50 +36,45 @@
         height: 300px;
         padding: 10px;
 
-        textarea.form-control {
-          border: 1px solid #cccccc;
-        }
+        .wrangler-plus-button {
```

Comment:
This should probably not be wrangler. Wrangler is a specific feature and this could be re-used across any webapp.